modules/swagger-codegen/src/main/resources/typescript-angular2/package.mustache

```diff
@@ -15,18 +15,24 @@
   "scripts": {
     "build": "typings install && tsc"
   },
+  "dependencies": {
```

Comment:
Shouldn't the next dependencies be peerDependencies? This give the consumer of the module more freedom.